src/raster/r.in.nasadem/r.in.nasadem.py

```diff
@@ -294,14 +295,13 @@ def download_tile(tile, url, pid, version, username, password):
 
         request = urllib2.Request(remote_tile)
         response = urllib2.urlopen(request)
-
         fo = open(local_tile, "w+b")
         fo.write(response.read())
         fo.close
         time.sleep(0.5)
-    except:
-        goturl = 0
-        pass
+    except urllib.error.URLError as err:
+        grass.fatal(_(f"Download of tile {local_tile} from URL {remote_tile} was not "
```

Comment:
Use `.format()` method instead of f-string (In particular, Python [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#f-strings) are not yet supported by `xgettext`.) for translation string, please.

Reply:
Thanks, I updated it accordingly